packages/@angular/cli/blueprints/class/index.ts

```diff
@@ -11,11 +13,18 @@ export default Blueprint.extend({
       name: 'spec',
       type: Boolean,
       description: 'Specifies if a spec file is generated.'
+    },
+    {
+      name: 'app',
```

Comment:
sure we should do it as a refactoring task as it needs to be done in multiple commands